Maple2.Server.Game/Packets/ItemScriptPacket.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using Maple2.Model.Game;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.Packets;
+
+namespace Maple2.Server.Game.Packets;
+
+public static class ItemScriptPacket {
+    private enum Command : byte {
+        LulluBox = 0,
+        TreasureChest = 4,
+        Gacha = 5,
+    }
+    
+    public static ByteWriter LulluBox(ICollection<Item> items) {
+        var pWriter = Packet.Of(SendOp.ItemScript);
+        pWriter.Write<Command>(Command.LulluBox);
+        pWriter.WriteInt(items.Count);
+        foreach (Item item in items) {
+            pWriter.WriteInt(item.Id);
+            pWriter.WriteInt(item.Amount);
+            pWriter.WriteInt(item.Rarity);
+            pWriter.WriteByte();
```

Comment:
im curious what this value is, can you try setting it to 1 and see if it displays any different? (might be a bool to display new or log in chat or something)

Reply:
ahhh this is just for the little sparkling animation around the item

set at false:
![false](https://i.imgur.com/1BPM7Q8.png)

![true](https://i.imgur.com/qR4oaQl.png)

Referencing this video it looks like it's always set to true regardless of the item
https://www.youtube.com/watch?v=laxvp4M8i3Y

Reply:
Can you add a comment for this